conf/app.ini

```diff
@@ -581,4 +581,4 @@ FILE_EXTENSIONS = .adoc,.asciidoc
 ; External command to render all matching extensions
 RENDER_COMMAND = "asciidoc --out-file=- -"
 ; Input is not a standard input but a file
-IS_INPUT_FILE = false
\ No newline at end of file
+IS_INPUT_FILE = false
```

Comment:
Also revert this change as it's not related to this PR

Reply:
This was an editor artifact but it's now reverted.  Every editor seems to introduce a final EOF new line character (vi, BBEdit, Atom, GithHub's built-in editor).  Finally had to use some perl to remove it: perl -pi -e 'chomp if eof' conf/app.ini.  Any reason why this file doesn't end with a new line?